frontend/src/main/web/lib/components/RecentContributions.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ContributionChart from './ContributionChart';
+import DropDown from './DropDown';
+import FilterableMatrixTable from './FilterableMatrixTable';
+import {DateRanges} from '../constants/Options';
+import Actions from '../actions/UserMatrixActions';
+
+var RecentContributions = React.createClass(
+  {
+    render: function() {
+      var dateRange = this.props.dateRange, chart = null, matrix = null;
```

Comment:
Also `var chart = null` seems almost identical to just `var chart` - technically `null` is different from `undefined` in javascript, but not in any way that makes a difference here.